rising/transforms/format.py

```diff
@@ -108,3 +108,21 @@ def __init__(self, keys: Union[Callable, Sequence], return_popped: bool = False)
 
     def forward(self, **data) -> Union[dict, Tuple[dict, dict]]:
         return filter_keys(data=data, keys=self.keys, return_popped=self.return_popped)
+
+
+class RenameKeys(AbstractTransform):
```

Comment:
For what is this trafo needed? For me it shouldn't matter which keys you specify.

Reply:
It is needed for external frameworks in the future. Generally speaking it is not necessary for us but might be a good transform to adjust rhe keys between a fixed dataset and already existing pipelines